cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/5_version_decouple.spec.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('Create datasources', () => {
+    before(() => {
+      // Clean up before creating new data sources for testing
+      cy.deleteAllDataSources();
+      cy.createDataSourceNoAuthWithTitle('ds_1');
+      cy.wait(6000);
+    });
+
+    after(() => {
+      // Clean up after all test are run
+      cy.deleteAllDataSources();
+      // remove the default data source
+      cy.setAdvancedSetting({
+        defaultDataSource: '',
```

Comment:
If you use the [singleDeleteDataSourceByTitle](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/dashboards/datasource-management-dashboards-plugin/commands.js#L288) , then you do no need to reset the defaultDataSource by updating the advancedSettings anymore